HOK.RibbonTab/HOK.RibbonTab/AppCommand.cs

```diff
@@ -53,6 +54,11 @@ public override void OnStartup()
             //CreateMissionControlPushButtons();
         }
 
+        private void OnDocumentSynchronizing(object sender, DocumentSynchronizingWithCentralEventArgs e)
+        {
+            HOK.Core.BackgroundTasks.Rules.AddAllSheetsToPrintSet(e.Document);
```

Comment:
Can we wrap this in a try/catch? Otherwise if there's an issue, it'll be shown to the user and we want this to be a background thing, even if it fails


```suggestion
            try
            {
                HOK.Core.BackgroundTasks.Rules.AddAllSheetsToPrintSet(e.Document);
            }
            catch (Exception ex)
            {
                Log.AppendLog(LogMessageType.EXCEPTION, ex.Message);
            }
```